### ISSUE TITLE
Deleted datasets return 500 error for file requests

### ISSUE THREAD

**Issue by franklin-feingold** [OpenNeuroOrg/openneuro#1162]
wanted to document these issues: 

a request for Google to validate fixes for:
'Duplicate without user-selected canonical'
'Server error (5xx)'

both returned that pages are affected by these

looking at the result for 'server error 5xx' it appears to be a few - ds 1128 and 1845
the 'duplicate without user-selected canonical' weirdly does not let me see because I do not have access (I have 2 used emails and neither have access)

**Comment by effigies** [comment]
These seem to have a spurious `crn/` in the path.

**Comment by nellh** [comment]
The `crn/` part is an artifact of having two APIs when the site first launched. One was under /api and the other was under /crn when OpenNeuro had not yet been named. These are indexed files Googlebot is trying to download, not routes in the React web app.

These are both deleted datasets, this error is because we serve a 500 instead of a 404 when the dataset and the file is deleted (vs 404 correctly when just the file is deleted).

**Comment by franklin-feingold** [comment]
Ahh thank you! 

So this appears it shouldn't be a cause of immediate concern? It can be scheduled and planned to serve the more correct error code

**Comment by nellh** [comment]
Yes, this is pretty low priority, it's not really an issue that a deleted dataset can't be search indexed other than the search console spam.

**Comment by franklin-feingold** [comment]
that sounds good! thank you!

**Comment by nellh** [comment]
I believe this was solved.
